pkg/registry/keychain_factory.go

```diff
@@ -7,23 +7,23 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
-type SecretRef struct {
+type ServiceAccountRef struct {
```

Comment:
This is super minor but, I still feel like it is a `SecretRef` because that is the ultimate thing that is getting pointed to.